coderd/userpassword/userpassword.go

```diff
@@ -138,10 +137,8 @@ func hashWithSaltAndIter(password string, salt []byte, iter int) string {
 // It returns properly formatted errors for detailed form validation on the client.
 func Validate(password string) error {
 	// Ensure passwords are secure enough!
-	// See: https://github.com/wagslane/go-password-validator#what-entropy-value-should-i-use
-	err := passwordvalidator.Validate(password, 52)
-	if err != nil {
-		return err
+	if len(password) < 6 {
```

Comment:
For now this logic stays pretty simple - here's the place where we'll be able to apply all the validation logic we want

Reply:
As far as I'm concerned here, we are squishing together three distinct separate changes:
1) Expose a password complexity validation endpoint on the backend
2) Modify the password validation logic on the FE to query the BE
3) **Modify the password validation logic on the BE**

Of these three options, 1) and 2) are relatively innocuous. 3) is the one I'm mainly concerned about.

I propose keeping the existing password validation logic (via entropy) in this PR and opening a separate PR to modify the password validation logic.